docs/docs/install/docker-compose/index.md

```diff
@@ -75,28 +76,33 @@
 
 1. Set up the web server.
 
+    !!! info "Database Configuration" 
+         By default, a Docker container named as `mathesar_db` running a Postgres server on internal port `5432` is created(it is not bound to the host, so it won't conflict with other services running on port `5432`).
+         Additionally, it comes with a default database and a superuser; this database can come in handy for storing Mathesar's [metadata](http://localhost:9000/install/configuration/#django_database_url).
+          The credentials for the Default database is
+           ```
+           DATABASE_NAME
+           USER='mathesar'
+           PASSWORD='mathesar'
+           ```
+        If you only plan to use Mathesar to connect to other preexisting Postgres database servers, then you may choose to [disable Mathesar's inbuilt Postgres database service](#disable-db-service) if you like.
+
     1. Edit your `.env` file, making the following changes:
 
         - Add the [**Backend Configuration** environment variables](../configuration.md#backend)
-        - Add the [**Database Configuration** environment variables](../configuration.md#database)
         - Customize the values of the environment variables to suit your needs.
 
         !!! example
             Your `.env` file should look something like this
             
             ``` bash
-            POSTGRES_USER='mathesar'
-            POSTGRES_PASSWORD='mathesar'
-            POSTGRES_PORT='5432'
             ALLOWED_HOSTS='https://<your_domain_name>'
             SECRET_KEY='dee551f449ce300ee457d339dcee9682eb1d6f96b8f28feda5283aaa1a21'
+            # For illustrative purposes, we will be using the credentials of the default database that is created automatically when the server starts.
```

Comment:
I don't like putting docs content in code blocks like this. It's easy for reader to miss. That's especially true in cases like this where the content is on a long line because mkdocs renders `<pre>` tags without soft line wrapping, meaning this sentence will be visually truncated. Can we put this content outside of the code block? Do we even need it at all? What's the purpose of including this sentence here?